config-service-impl/src/main/java/org/hypertrace/config/service/store/FilterBuilder.java

```diff
@@ -106,4 +100,39 @@ private Filter evaluateLeafExpression(RelationalFilter relationalFilter) {
   private String buildConfigFieldPath(String configJsonPath) {
     return String.format("%s.%s", CONFIG_FIELD_NAME, configJsonPath);
   }
+
+  public Object convertValueToObject(Value value) {
```

Comment:
I wonder if we need these / already have them?

In the doc write flow, we have the implementer convert between proto and value. The proto serializes to a json string which is basically what we want here. In other words:
- does this belong in `ConfigProtoConverter` where we already have Value->Proto,  Proto->Value, Value->String, String->Value methods?
- If yes, could we just do Value->String and then toss it in object mapper to convert it like this?